packages/env/lib/init-config.js

```diff
@@ -169,10 +169,12 @@ function installXdebug( enableXdebug ) {
 
 	return `
 # Install Xdebug:
+RUN apt-get update -y && apt-get -y install iproute2
 RUN if [ -z "$(pecl list | grep xdebug)" ] ; then pecl install xdebug ; fi
 RUN docker-php-ext-enable xdebug
 RUN echo 'xdebug.start_with_request=yes' >> /usr/local/etc/php/php.ini
 RUN echo 'xdebug.mode=${ enableXdebug }' >> /usr/local/etc/php/php.ini
 RUN echo '${ clientDetectSettings }' >> /usr/local/etc/php/php.ini
+RUN HOST_IP=$(ip route | awk '/default/ { print $3 }'); echo "xdebug.client_host=\"$HOST_IP\"" >> /usr/local/etc/php/php.ini
```

Comment:
What does this change do?

Reply:
When PHPUnit is used, xdebug is not able to detect host ip within the docker container. It can do it through the browser because of some HTTP fields, but that doesn't exist in the CLI. The host IP is the "default gateway" within the docker container, which can be seen using the `ip route` command. This extracts the default gateway from the `ip route` command and sets it as host ip.